index.bs

```diff
@@ -11302,6 +11306,181 @@ The [=remote end steps=] given |session| and |command parameters| are:
 </div>
 
 
+## The webExtension Module ## {#module-webExtension}
+
+The <dfn export for=modules>webExtensions</dfn> module contains functionality for
+managing and interacting with web extensions.
+
+### Definition ### {#module-webExtensions-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+WebExtensionsCommand = (
+  webExtensions.Install,
+  webExtensions.Uninstall
+)
+</pre>
+
+[=local end definition=]
+
+<pre class="cddl local-cddl">
+WebExtensionsResult = (
+  webExtensions.InstallResult
+)
+</pre>
+
+### Types ### {#module-webExtensions-types}
+
+#### The webExtensions.Extension Type #### {#type-webExtensions-Extension}
+
+<pre class="cddl remote-cddl local-cddl">
+webExtensions.Extension = text
+</pre>
+
+The <code>webExtensions.Extension</code> type represents a web extension id within a browser.
+
+### Commands ### {#module-webExtensions-commands}
+
+#### The webExtensions.install Command #### {#command-extensions-install}
+
+The <dfn export for=commands>webExtensions.install</dfn> command installs a web extension for the session.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      webExtensions.Install = (
+         method: "webExtensions.install",
+         params: webExtensions.InstallParameters
+      )
+
+      webExtensions.InstallParameters = {
+         extensionData: webExtensions.ExtensionData,
+         ? allowPrivateBrowsing: bool .default false,
```

Comment:
as was suggested in other comments, let's drop allowPrivateBrowsing and related steps as this concept is not well defined in the specification and might not be supported by all implementations.